src/RequestAPI.ts

```diff
@@ -191,6 +196,45 @@ export async function renameResource(
     }
 }
 
-export async function getFrame(idx: Number): Promise<boolean> {
-    return false;
+
+/**
+ * get all the chapters of the corresponding project
+ * @param name: project_name
+ * @returns
+ */
+export async function getChapters(name: string): Promise<string[]> {
+    // need to update to correct function
+    return ["test chapter","next is chapter name",name, "end of chapter" ];
+}
+
+/**
+ * get all the frames of the corresponding chapter
+ * @param name: project_name
+ * @param chapter_name: chapter_name
+ * @returns
+ */
+export async function getFrames(name: string, chapter_name: string): Promise<IFrame[]> {
```

Comment:
Consider adding error handling to make it more robust

Reply:
waiting for back end to finish